lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -93,6 +93,34 @@ function M.select_compiler_args(repo, compiler)
   end
 end
 
+function M.select_compile_command(repo, cc, compile_location)
+  if string.match(cc, "cl$") or string.match(cc, "cl.exe$") or not repo.use_makefile or fn.has "win32" == 1 then
+    return {
+      cmd = cc,
+      info = "Compiling...",
+      err = "Error during compilation",
+      opts = {
+        args = vim.tbl_flatten(M.select_compiler_args(repo, cc)),
```

Comment:
I feel like we should also use the `cxx_standard` key in the `select_compiler_args` for completeness too, do you think this is wise ?

Reply:
If it works without C++14, I would leave it that way. I think only Haskell and Norg require C++14 at the moment which works with a recent Windows compiler or Makefiles on Linux/MacOS

Reply:
Ah, now I remember. It is not possible when invoking a C compiler with a single invocation. This is why I was using the Makefiles. In theory, it should work when you scope the cxx_standard with `-Xc++` flag. But it didn't work when I tried it.